### ISSUE TITLE
Text indenting does not work in Live Share client although it was set as smart.

### ISSUE THREAD

**Issue by zhouwangyang** [microsoft/PTVS#6149]
**Steps to Reproduce**
1.Make connection with Live Share in Python application.
2.Make sure  the smart indent option was set as smart both in host and client.
(tools/options/text editor/python/tabs/Indenting)
3.Typying code "try:" in client, and the press enter after the colon.

**Expected behavior**
the cursor should automatic indent stops at four Spaces.

**Actual behavior**
the cursor stops in the beginning of the line. work for me in the host, but failed in client.  
![image](https://user-images.githubusercontent.com/39871640/84884378-3a838d80-b0c4-11ea-9651-c75646f7c92f.png)

**build**
![image](https://user-images.githubusercontent.com/39871640/84884432-4a9b6d00-b0c4-11ea-9b28-f5772fe4830e.png)

**Comment by huguesv** [comment]
@zhouwangyang is this a regression?

**Comment by zhouwangyang** [comment]
@huguesv  no, it is not a regression. it was the first time to be found in Live Share client.
